examples/llm_plus_gnn/rag_generate_multihop.py

```diff
@@ -0,0 +1,48 @@
+# %%
+import pandas as pd
+import torch
+import tqdm
+from profiling_utils import create_remote_backend_from_triplets
+from rag_feature_store import SentenceTransformerFeatureStore
+from rag_graph_store import NeighborSamplingRAGGraphStore
+
+from torch_geometric.datasets.updated_web_qsp_dataset import (
+    preprocess_triplet,
+    retrieval_via_pcst,
+)
+from torch_geometric.loader import RAGQueryLoader
+from torch_geometric.nn.nlp import SentenceTransformer
+
+# %%
+triplets = torch.load('wikimultihopqa_full_graph.pt')
```

Comment:
same thing, move to examples/llm_plus_gnn/data_handling

Reply:
u have moved this to benchmark folder but what is being benchmarked? i dont see metrics being tracked.

Reply:
also in general, the real benchmarks belong here: https://github.com/pyg-team/pytorch_geometric/tree/master/benchmark
not inside the examples folder